v03_pipeline/lib/reference_datasets/misc.py

```diff
@@ -45,6 +47,16 @@ def get_enum_select_fields(
     return enum_select_fields
 
 
+def mito_contig_filter(
+    reference_genome: ReferenceGenome,
+    dataset_type: DatasetType,
+    ht: hl.Table,
+) -> Callable[[DatasetType, ReferenceGenome, hl.Table], hl.Expression]:
```

Comment:
any reason to return a callable instead of calling `hl.filter` inside of the function? (I'm also wondering if I should do this for the custom select)

Reply:
I don't think there is!  We're doing that with `filter_contigs`, so we should be consistent with that (or make `filter_contigs` consistent with this, but I think changing it makes it just ever so slightly clearer)!